docs/LinkingLibraries.md

```diff
@@ -24,7 +24,41 @@ error as soon as you try to use the library._
 
 ## Here the few steps to link your libraries that contain native code
 
-### Step 1
+### Automatic linking
+
+There is a way to link your native dependencies automatically by using [rnpm](http://github.com/rnpm/rnpm):
```

Comment:
Reword: `"[rnpm](http://github.com/rnpm/rnpm)" is a community project that allows linking of native dependencies automatically:"